### PR TITLE
Added html for section 1

### PR DESCRIPTION
**Pull request by Aleksey255**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            <a class="link" href="https://google.com">
+              <img src="assets/icons/google.svg" alt="google icon">
+            </a>
+          </div>
+        </div>
+        <div class="img-block">
+          <img src="assets/images/image-1.jpeg" alt="bag and device">
+        </div>
+      </div>
+    </section>
```
Comment (nagorapro) [inline index.html:30]: `a + b`

Comment (nagorapro) [inline index.html:30]: Use BEM methodology
Use "download" class name
Use "download" id name

**Review by nagorapro — COMMENTED** [review]
Need fix

**Review by Palych18 — APPROVED** [review]
Done

**Pull request merged**